packages/@react-native-windows/cli/src/postinstall.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as chalk from 'chalk';
+import {rm, cp} from 'shelljs';
```

Comment:
Should we use 'fs' or 'fsextra' instead of shell?

Reply:
We don't have an existing direct dependency on fs-extra (though I would guess a subdependency will), but fs doesn't have a method to copy recursive. Used existing shelljs dependency.